components/atoms/Stages.js

```diff
@@ -1,36 +1,48 @@
-import Image from "next/image";
-import Link from "next/link";
 import PropTypes from "prop-types";
-
+import { useState, useRef } from "react";
 /**
  *  Stages Component
  */
 
 export default function Stages(props) {
+  // const { items } = useContext(LanguageContext);
+  // const language = items.language;
+  // const t = language === "en" ? en : fr;
+
+  const [showFlag, setShowFlag] = useState(false);
+  const subJourneyData = useRef();
+
+  function onClickHandler(data) {
+    subJourneyData.current = data;
+    setShowFlag(true);
+  }
+
   return (
     <div className="bg-image flex flex-col sm:flex-wrap sm:flex-row justify-center border border-black">
-      {props.journeys.map((journey, idx) => (
-        <div
-          key={idx}
-          className="h-46px card my-6 mx-4 pt-3 rounded font-body text-sm text-center text-white cursor-pointer border journeyButton"
-        >
-          {journey.title}
+      {showFlag === false ? (
+        props.journeys.map((journey, idx) => (
+          <div
+            key={idx}
+            className="h-46px card my-6 mx-4 pt-3 rounded font-body text-sm text-center text-white cursor-pointer border journeyButton"
+            onClick={() => onClickHandler(journey.subJourney)}
+          >
+            {journey.title}
+          </div>
+        ))
+      ) : (
+        <div className="subJourneys">
```

Comment:
Saw this in both `Stages` and `StagesV2`, but can't find this in any of the css?